tests/unit/adapters/drf-test.js

```diff
@@ -33,6 +33,7 @@ test('buildURL', function(assert) {
   var adapter = this.subject();
   assert.equal(adapter.buildURL('Animal', 5, null), 'test-host/test-api/animals/5/');
   assert.equal(adapter.buildURL('FurryAnimals', 5, null), 'test-host/test-api/furry-animals/5/');
+  assert.equal(adapter.buildURL('Animal', null, null, 'query', { limit: 10 }), 'test-host/test-api/animals/');
```

Comment:
FWIW the following code in my application did produce a url with a querystring at the end using ember-data 1.0.0-beta.18 so I'm wondering if it's a version issue?

 ```
DRFAdapter.reopen({
    buildURL: function(type, id, snapshot, requestType, queryParams) {
      var url = this._super(type, id, snapshot, requestType, queryParams);
      if (url.charAt(url.length - 1) !== '/') {
        url += '/';
      }
      return url;
    }
  });
```